syntax = "proto3";

package protos;

service MonitorService {
    rpc Heartbeat (HeartbeatRequest) returns (HeartbeatOkResponse) {}
    rpc InstanceMetrics (InstanceMetricsRequest) returns (InstanceMetricsOkResponse) {}
}

message HeartbeatRequest {}

message HeartbeatOkResponse {}

message InstanceMetricsRequest {}

message InstanceMetricsOkResponse {
    float cpu_load = 1;
}